packages/database/test/helpers/syncpoint-util.ts

```diff
@@ -366,10 +368,14 @@ export class SyncPointTestParser {
           q = query(q, limitToFirst(paramValue));
         } else if (paramName === 'limitToLast') {
           q = query(q, limitToLast(paramValue));
+        } else if (paramName === 'startAfter') {
+          q = query(q, startAfter(paramValue.index, paramValue.name));
         } else if (paramName === 'startAt') {
           q = query(q, startAt(paramValue.index, paramValue.name));
         } else if (paramName === 'endAt') {
           q = query(q, endAt(paramValue.index, paramValue.name));
+        } else if (paramName === 'endBefore') {
+          q = query(q, endBefore(paramValue.index, paramValue.name));
```

Comment:
It looks like we don't have any tests that use the `paramValue.name`, could you add some?

Reply:
Done.